server/resource_server/constants/verification.go

```diff
@@ -0,0 +1,7 @@
+package constants
+
+import "time"
+
+const VerificationCodeValidityDuration = time.Minute * 2
+const VerificationEmailSubject = "Verify your email at the Layer8 service"
+const VerificationCodeSize = 6
```

Comment:
can we make this configurable via env variable?

Reply:
Yes, sounds good. Will do that.